source/cloud-security/amazon/services/supported-services/waf.rst

```diff
@@ -10,9 +10,58 @@ Amazon Web Application Firewall (WAF)
 
 `Amazon WAF <https://aws.amazon.com/waf/>`_ is a web application firewall that helps protect your web applications from common web exploits that could affect application availability, compromise security, or consume excessive resources. ``AWS WAF`` gives you control over which traffic to allow or block to your web applications by defining customizable web security rules. 
 
-Amazon configuration
+Amazon configuration native integration
+---------------------------------------
+
+#. :doc:`Create a new </cloud-security/amazon/services/prerequisites/S3-bucket>` S3 bucket. (If you want to use an already created one, skip this step).
+
+#. Go to Services > Security, Identity, & Compliance > WAF & Shield
+
+    .. thumbnail::  /images/aws/waf-native-1.png
+      :align: center
+      :width: 70%
+
+#. Click *Create web ACL*
+
+    .. thumbnail::  /images/aws/waf-native-2.png
+      :align: center
+      :width: 70%
+
+#. Put a name to your web ACL and click on the *Next* button at the bottom of the page:
```

Comment:
```suggestion
#. Choose a name for your Web ACL and click on *Next*:
```

The next button is not shown in the image, perhaps we could do a little of zoom out and include it to be clear.